src/time_zone_format.cc

```diff
@@ -500,11 +516,7 @@ const char* ParseSubSeconds(const char* dp,
 // Parses a string into a std::tm using strptime(3).
 const char* ParseTM(const char* dp, const char* fmt, std::tm* tm) {
   if (dp != nullptr) {
-#if defined(__MINGW32__) || defined(__MINGW64__)
-    dp = nullptr;
-#else
-    dp = strptime(dp, fmt, tm);
-#endif    
+    dp = RcppCCTZ::strptime(dp, fmt, tm);
```

Comment:
Maybe standard `strptime()` most of the time and our if and when we have no alternative?